convert2rhel/utils.py

```diff
@@ -538,8 +538,9 @@ def require_single_user_mode():
     """
     loggerinst = logging.getLogger(__name__)
 
-    output, ret_code = run_cmd_in_pty("runlevel", print_cmd=False)
-    runlevel = output.replace("\r\n", "")[-1]
+    runlevel_path = pexpect.which("runlevel")
+    output, ret_code = run_subprocess(cmd=runlevel_path, print_cmd=False)
```

Comment:
```suggestion
    output, ret_code = run_subprocess(cmd=runlevel_path, print_cmd=True, print_output=False)
```
See https://github.com/oamg/convert2rhel/pull/148.